src/lib/homestore/heap_chunk_selector.cpp

```diff
@@ -117,4 +117,16 @@ void HeapChunkSelector::build_per_dev_chunk_heap(const std::unordered_set< chunk
         if (excludingChunks.find(p.first) == excludingChunks.end()) { add_chunk_internal(p.first); }
     };
 }
-} // namespace homeobject
\ No newline at end of file
+
+homestore::blk_alloc_hints HeapChunkSelector::get_blk_alloc_hints_on_same_pdev(chunk_num_t chunk_id) const {
+    auto iter = m_chunks.find(chunk_id);
+    if (iter == m_chunks.end()) {
+        LOGWARN("No chunk found for ChunkID {}, will return default blk alloc hints", chunk_id);
```

Comment:
Should we use the `homeobject` logging module?

Reply:
yes, I just noticed this interface and had updated to use it.